controllers/addons_controller.go

```diff
@@ -308,3 +325,22 @@ func indexHelmReleaseByOwner(o runtime.Object) []string {
 
 	return []string{owner.Name}
 }
+
+func indexHelmRepoByOwner(o runtime.Object) []string {
+	reconciler.Log.V(1).Info("processing", "helmrepositories.source.toolkit.fluxcd.io", utils.LogJSON(o))
```

Comment:
This should probably also log as "indexing" rather than "processing" and also at a higher verbosity level than Info or that standard Debug as it'a likely to be chatty.

Reply:
done